### ISSUE TITLE
Images in lists question?

### ISSUE THREAD

**Issue by casperl** [theNewDynamic/gohugo-theme-ananke#160]
Using the `featured_image = ""` directive in the page header for a blog post etc causes the image on top of the page to change to that image as well as an image on the lists of new posts on the front page, though that does not work for subsequent sub-directories.  To illustrate this graphically:

( http://bitterbek.co.za/ ) Site front page ( content/ ) = list with [image]+[page summary]
( http://bitterbek.co.za/skrywes/ ) Sub directories under content ( content/skrywes ) = list with [page summary] <-- no images next to content.

Question: how can I get the page lists on sub directory pages to display [image]+[page summary] ?

I have an _index.md file in the content root as well as subdirs to convey whatever is specific to that category, with the page list of sub-pages appearing below that.  I cannot spot a difference that will cause this behavior in my _index.md files or elsewhere.  What am I missing here?

**Comment by budparr** [comment]
Hey, @casperl So if I understand you, you want this (on http://bitterbek.co.za/skrywes/):

![screen shot 2019-03-02 at 2 00 36 pm](https://user-images.githubusercontent.com/115347/53686334-9ec5e680-3cf3-11e9-8fcd-4f23e2d6f33a.jpg)

to look like this (home page):

![screen shot 2019-03-02 at 2 00 17 pm](https://user-images.githubusercontent.com/115347/53686342-b0a78980-3cf3-11e9-9d2f-2599e5bb6965.jpg)

**Comment by casperl** [comment]
Yes, exactly. I think I found it, in ananke/layouts/index.html there is a partial called {{ partial "summary-with-image.html" . }} that is obviously invoked on the front page.  

At this stage I am guessing I have to create a custom index.html file, similar to that referenced above in either mysite/layouts/index.html of a dir structure mimicking my content structure mysilte/layouts/skrywes/index.html that contains {{ partial "summary-with-image.html" . }}

I have learned the hard way not to interfere by placing custom stuff into the theme folders specifically, so I presume mysite/layouts is the right place for custom modifications.

Work interferes, so I cannot experiment now, but I will attempt this at home tonight.

Thanks for a magnificent theme!

**Comment by casperl** [comment]
I tried the above with now success.  Back to reading the Hugo documentation on themes, though I keep on getting the feeling I am missing something important that I simply don't find in what I am reading.

**Comment by budparr** [comment]
I'll have a look this weekend

**Comment by casperl** [comment]
I've achieved it by creating a shortcode that shows the page image directly above the summary as well as changing index.html to call summary and not summary-with-image.  On single pages the same image is duplicated, though it does not look bad at all.

![20190417-142427](https://user-images.githubusercontent.com/1303185/56287529-e1654600-611c-11e9-8755-d45b833ccc32.png)
![20190417-142740](https://user-images.githubusercontent.com/1303185/56287558-f9d56080-611c-11e9-8cba-7cfd80387381.png)
![20190417-143240](https://user-images.githubusercontent.com/1303185/56287840-b29b9f80-611d-11e9-9ab8-9f702b9b3865.png)

Blog - http://bitterbek.co.za

**Comment by casperl** [comment]
I neglected to mention that featured_image files are resized to 640x300 res that works across the board. Other sizes produce weird effects.  

Here is the code for my "masthead.html" shortcut, no rocket science though it may save some time for implementing this feature.
```
<a href="{{.Page.Permalink}}" class="link">
<img src="{{ .Page.Params.featured_image }}" 
     style="display: block; 
            margin-left: auto; 
            margin-right: auto;">
</a>
```
A possible modification I may do (once I have auquired the skills) is to include this code in the templates for Lists and Single pages but not on Index.html.  And then a switch in .Page.Params that enables/disables this modification in the theme.  Something in config.toml like 

```
[Params]
  masthead = true # variable to enable the displays of .Params.featured_image on Lists and Single pages.
```
